views/demo-flows.json

```
@@ -0,0 +1,34 @@
+{
+    "prefect-recipes": {
+        "hello": {
+            "description": {
+                "summary": "A simple flow that says hello.",
+                "returns": "",
+                "examples": []
+            },
+            "documentation_url": "https://github.com/PrefectHQ/prefect-recipes",
+            "entrypoint": "prefect_airbyte/flows-simple/hello.py:hello",
```

This depends on how it will be run, so I don't know if the suggestion is correct, but prefect_airbyte should be removed or replaced.
```suggestion
            "entrypoint": "flows-simple/hello.py:hello",
```